.github/workflows/go.yml

```diff
@@ -11,21 +11,21 @@ on:
 jobs:
   lint:
```

Comment:
does this need to be `go-fmt` for dependabot to inspect it?

Reply:
I don't think dependabot matches based off the job name. At least, I don't' see anything in `dependabot.yml` that indicates that.